common/internal_model/src/main/scala/uk/ac/wellcome/models/marc/MarcLanguageCodeList.scala

```diff
@@ -0,0 +1,98 @@
+package uk.ac.wellcome.models.marc
+
+import grizzled.slf4j.Logging
+
+import scala.xml.XML
+
+// Utilities for parsing the MARC Code List for Languages to looking up
+// language labels.
+//
+// This parses the XML file downloaded from:
+// https://www.loc.gov/standards/codelists/languages.xml
+//
+object MarcLanguageCodeList extends Logging {
+
+  // Create a lookup from code -> name
+  private val codeLookup: Map[String, String] = {
```

Comment:
Might be worth making these `lazy`? Otherwise ever project that uses internal model has to load that huge XML file.

Reply:
Done.